flight/Modules/Airspeed/airspeed.c

```diff
@@ -176,7 +176,7 @@ MODULE_INITCALL(AirspeedInitialize, AirspeedStart);
  */
 static void airspeedTask(void *parameters)
 {
-	AirspeedSettingsUpdatedCb(AirspeedSettingsHandle());
+	AirspeedSettingsUpdatedCb(NULL, NULL, NULL, 0);
```

Comment:
The unfortunate thing is that it's the per-module callback function, so adding a wrapper in each module doesn't necessarily make it cleaner IMO.

An alternative would be to make the registration functions guarantee to generate at least one callback quasi-immediately so that we don't need to do it explicitly.  It would also remove race conditions-- since this could be running "at the same time" with a different version of the data if tasks switch at the wrong time.